docs/src/references/c8y-configuration-management.md

```diff
@@ -122,7 +237,7 @@ OPTIONS:
         --config-dir <CONFIG_DIR>      [default: /etc/tedge]
         --debug                        Turn-on the debug log level
     -h, --help                         Print help information
-    -i, --init                         Create supported operation files
+    -i, --init [CHILD]                 Create supported operation files, possibly for several child devices
```

Comment:
What is "CHILD"? It is the child-id, or as you used before in the document $CHILD_DEVICE_ID.

Reply:
Updated